dts/bindings/wifi/infineon,cyw43xxx-wifi-sdio.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2023 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    CYW43xxx WIFI Connectivity.
+
+compatible: "infineon,cyw43xxx-wifi-sdio"
+
+include: [base.yaml, pinctrl-device.yaml]
+
+properties:
+  wifi-reg-on-gpios:
+    type: phandle-array
+
+  wifi-host-wake-gpios:
+    type: phandle-array
+
+  wifi-dev-wake-gpios:
+    type: phandle-array
+
+  wifi-sdio-cmd-gpios:
+    type: phandle-array
+    required: true
+
+  wifi-sdio-clk-gpios:
+    type: phandle-array
+    required: true
+
+  wifi-sdio-d0-gpios:
+    type: phandle-array
+    required: true
+
+  wifi-sdio-d1-gpios:
+    type: phandle-array
+    required: true
+
+  wifi-sdio-d2-gpios:
+    type: phandle-array
+    required: true
+
+  wifi-sdio-d3-gpios:
+    type: phandle-array
+    required: true
```

Comment:
I'm not sure it make sense to prepend all properties with `wifi-`. IMO they give no value and could be dropped. This binding is only for WiFi part, right?

Please also provide some documentation, similar to https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/bluetooth/infineon%2Ccyw43xxx-bt-hci.yaml

Reply:
done.